pkg/apis/core/validation/validation_test.go

```diff
@@ -5651,21 +5651,16 @@ func TestValidatePodSpec(t *testing.T) {
 	minGroupID := int64(0)
 	maxGroupID := int64(2147483647)
 
-	priorityEnabled := utilfeature.DefaultFeatureGate.Enabled("PodPriority")
-	defer func() {
-		var err error
+	defer func(podPriority, shareProcessNamespace bool) {
```

Comment:
utilfeaturetesting.SetFeatureGateDuringTest will do this for you

Reply:
egad, I've been wanting that for like a year. I wish I had sent you a PR sooner. Fixed.